aws/table_aws_accessanalyzer_finding.go

```diff
@@ -0,0 +1,281 @@
+package aws
+
+import (
+	"context"
+	// "strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/accessanalyzer"
+	"github.com/aws/aws-sdk-go-v2/service/accessanalyzer/types"
+
+	accessanalyzerv1 "github.com/aws/aws-sdk-go/service/accessanalyzer"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+type accessanalyzerFindingInfo = struct {
+	Finding types.Finding
+	AccessAnalyzerArn     string
+}
+
+func tableAwsAccessAnalyzerFinding(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_accessanalyzer_finding",
+		Description: "AWS Access Analyzer Finding",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"id", "access_analyzer_arn"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "ValidationException", "InvalidParameter"}),
+			},
+			Hydrate: getAccessAnalyzerFindings,
+			Tags:    map[string]string{"service": "access-analyzer", "action": "GetFinding"},
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listAccessAnalyzers,
+			Hydrate:       listAccessAnalyzersFindings,
```

Comment:
Should not we handle the not found error code for list config?
I can see the [ListFindings](https://docs.aws.amazon.com/access-analyzer/latest/APIReference/API_ListFindings.html) API throws `ResourceNotFoundException` if the specified resource could not be found.